docs: Updates to Modules chapter

jwwojak opened the pull request:
# Overview

This PR updates the Flex Instruction Manual, Chapter 4, Modules. New content for:

- Plate reader
- HEPA/UV module
- Compatibility table
- Small revisions throughout

Branch is `docs-flex-manual-plate-reader`

Original drafts with writer and reviewer comments are here: [Modules Chapter: New modules](https://docs.google.com/document/d/1a8YXvt3lSMhxsT_lp3OJ7QsQytVEo9msAdcDtr-fYKc/edit?usp=sharing).

JIRA

- RTC-588: HEPA/UV module
- RTC-589: Plate reader
- RTC-590: Module/robot compatibility table

## Test Plan and Hands on Testing

Prose content previously reviewed here: [Modules Chapter: New modules](https://docs.google.com/document/d/1a8YXvt3lSMhxsT_lp3OJ7QsQytVEo9msAdcDtr-fYKc/edit?usp=sharing).

Can't hurt to check the writing and the coding.

## Changelog

Chapter 4, Modules, new and revised content for

- Plate reader
- HEPA/UV module
- Compatibility table
- Small intro revisions
- New images for the plate reader and HEPA/UV module

Also, found and fixed a small typo in Ch. 6. 

## Review requests

None

## Risk assessment

Low: documentation

jwwojak committed:
Modules intro and plate reader

Small changes to the chapter intro.

Revise compatibility table.

Add new plate reader section.

jwwojak committed:
HEPA-UV module push

Adds new section for this module.

jwwojak committed:
Merge branch 'edge' into docs-flex-manual-plate-reader

jwwojak committed:
Merge branch 'edge' into docs-flex-manual-plate-reader

ecormany commented on docs/flex-manual/docs/modules.md:
```diff
@@ -18,8 +28,12 @@ Opentrons Flex is compatible with four types of on-deck Opentrons modules:
 
 Some modules originally designed for the OT-2 are compatible with Flex, as summarized in the table below. A checkmark indicates compatibility, and an X indicates incompatibility.
 
+<!-- observation: the X marks in the table are very small in the browser.
```
Yes, I think the two choices here are:
- Emoji ✅ and ❌ 
- Material for Mkdocs icons

The former give less control, but make for highly readable source. The latter let us do just about whatever we want in terms of variant, color, etc.

jwwojak replied:
Trying some Material icons:

:heavy_check_mark: or `:heavy_check_mark:`

:heavy_multiplication_x: or `:heavy_multiplication_x:`

ecormany commented on docs/flex-manual/docs/modules.md:
```diff
@@ -29,6 +43,7 @@ Some modules originally designed for the OT-2 are compatible with Flex, as summa
 | Thermocycler Module GEN1        |  ✓   |  ×   |
 | Thermocycler Module GEN2        |  ✓   |  ✓   |
 | HEPA Module                     |  ✓   |  ×   |
+| HEPA/UV Module                  |  ×   |  ✓   |
```
Not really sure why HEPA was at the bottom. We should fully alphabetize the table.

jwwojak replied:
Will put table in alpha order.

ecormany commented on docs/flex-manual/docs/modules.md:
```diff
@@ -71,7 +86,9 @@ Thermocycler Modules.
-Module calibration is required for all modules that install via a caddy: the Heater-Shaker, Temperature, and Thermocycler Modules. The Magnetic Block doesn't require calibration, and is ready for use as soon as you place it on the deck.
+Calibration is required for some modules that use a separate caddy, specifically the Heater-Shaker, Temperature, and Thermocycler Modules.
+
+Other modules do not require calibration and are ready for use upon installation. These include the Absorbance Plate Reader (which ships preinstalled in its caddy), the HEPA/UV Module, and the Magnetic Block
```
```suggestion
Other modules do not require calibration and are ready for use upon installation. These include the Absorbance Plate Reader (which ships preinstalled in its caddy), the HEPA/UV Module, and the Magnetic Block.
```

ecormany commented on docs/flex-manual/docs/modules.md:
```diff
@@ -95,6 +112,115 @@ Once calibration is complete and you've removed the adapter and probe, the modul
+
+#### Measurement capabilities
+
+The Plate Reader uses 96 separate detection units for rapid sample analysis. The detection units use light in the 400–700 nanometer (nm) range to determine sample concentrations.
```
We did `plate reader` lowercase in subsequent mentions in the print docs, and should follow suit here.

jwwojak replied:
Ok. Changing to "plate reader" for subsequent mentions.

ecormany commented on docs/flex-manual/docs/protocol-development.md:
```diff
@@ -94,13 +94,11 @@ In general, following the style of the methods section of an academic paper will
 #### Custom protocol pricing
 
-Custom Protocol Developmentis a service available to all owners of Opentrons Flex systems. Opentrons provides remote and onsite protocol development services customized to your specific workflow. Price and development time are based on the complexity of your protocol and the related code. See the [Instrument Services section](https://opentrons.com/instrument-services) of the Opentrons website to  ...
+Custom Protocol Development is a service available to all owners of Opentrons Flex systems. Opentrons provides remote and onsite protocol development services customized to your specific workflow. Price and development time are based on the complexity of your protocol and the related code. See the [Instrument Services section](https://opentrons.com/instrument-services) of the Opentrons website to ...
```
Good catch! And a good reminder that I haven't fully proofread the manual REV3 content and will need to do so before launch.

For next time, please make a separate PR for totally separate content. But no need to remove it from here now.

jwwojak replied:
Accidental discovery while checking an anchor link. Will make separate PR going forward.

ecormany reviewed (commented):
This looks great. Seamless addition to the chapter. mkdocs formatting is perfect. Only a few typo-level changes to fix before this is ready to merge.

jwwojak committed:
Merge branch 'edge' into docs-flex-manual-plate-reader

jwwojak committed:
Update docs/flex-manual/docs/modules.md

Committing recommendations. I forgot a period.

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
Modules revisions

- Put Flex / OT-2 compatibility table in alpha order.

- Change subsequent mentions of "Plate Reader" to "plate reader"

- Change check mark and X symbols in compatibility table to use larger icons from the material library.

ecormany commented on docs/flex-manual/docs/modules.md:
```diff
@@ -26,24 +26,22 @@ Opentrons Flex is compatible with with the following Opentrons modules:
-| Thermocycler Module GEN2        |  ✓   |  ✓   |
-| HEPA Module                     |  ✓   |  ×   |
-| HEPA/UV Module                  |  ×   |  ✓   |
+Some modules originally designed for the OT-2 are compatible with Flex, as summarized in the table below. A checkmark :heavy_check_mark: indicates compatibility, and an :heavy_multiplication_x: indicates incompatibility.
```
nice, these symbols are very clear. we can color them later if we want.

ecormany commented on docs/flex-manual/docs/modules.md:
```diff
@@ -88,7 +86,7 @@ Thermocycler Modules.
 Calibration is required for some modules that use a separate caddy, specifically the Heater-Shaker, Temperature, and Thermocycler Modules.
 
-Other modules do not require calibration and are ready for use upon installation. These include the Absorbance Plate Reader (which ships preinstalled in its caddy), the HEPA/UV Module, and the Magnetic Block
+Other modules do not require calibration and are ready for use upon installation. These include the plate reader (which ships preinstalled in its caddy), the HEPA/UV Module, and the Magnetic Block.
```
This should match the other items in these lists, which use full names.
```suggestion
Other modules do not require calibration and are ready for use upon installation. These include the Absorbance Plate Reader Module (which ships preinstalled in its caddy), the HEPA/UV Module, and the Magnetic Block.
```

ecormany commented on docs/flex-manual/docs/modules.md:
```diff
@@ -95,6 +110,115 @@ Once calibration is complete and you've removed the adapter and probe, the modul
+
+![plate reader hero](images/plate-reader-hero-lid-off.png)
+
+### Plate Reader Features
```
Just spotted this: sentence case for headers. Will add suggestions here and on other occurrences.
```suggestion
### Plate reader features
```

ecormany commented on docs/flex-manual/docs/modules.md:
```diff
@@ -95,6 +110,115 @@ Once calibration is complete and you've removed the adapter and probe, the modul
+
+The plate reader fits in deck slots A3–D3 only. It comes preinstalled in a caddy, which helps secure the unit to the deck. This module does not require calibration, but you can run Labware Position Check on any installed labware.
+
+#### Software Control
```
```suggestion
#### Software control
```

ecormany commented on docs/flex-manual/docs/modules.md:
```diff
@@ -95,6 +110,115 @@ Once calibration is complete and you've removed the adapter and probe, the modul
+
+The plate reader is fully programmable in Protocol Designer and the Python Protocol API.
+
+### Plate Reader Specifications
```
```suggestion
### Plate reader specifications
```

ecormany commented on docs/flex-manual/docs/modules.md:
```diff
@@ -118,26 +116,26 @@ At any time, you can view and manage your module calibration data in the Opentro
 
 !!!note
-    The Opentrons Flex Absorbance Plate Reader may currently not be offered, used or put on the market in any European Patent Convention States due to a third-party patent application.
+    The plate reader may currently not be offered, used or put on the market in any European Patent Convention States due to a third-party patent application.
```
Use full name. This is legal text and needs to be exact.
```suggestion
    The Opentrons Flex Absorbance Plate Reader Module may currently not be offered, used or put on the market in any European Patent Convention States due to a third-party patent application.
```

ecormany reviewed (commented):
Just a few final proofread changes. I've left suggestions for all of them so you should be able to batch them up as a final commit.

jwwojak committed:
Update docs/flex-manual/docs/modules.md

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
Update docs/flex-manual/docs/modules.md

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
Update docs/flex-manual/docs/modules.md

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
Update docs/flex-manual/docs/modules.md

Change header case

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
Update docs/flex-manual/docs/modules.md

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
Merge branch 'edge' into docs-flex-manual-plate-reader
